File: CS/DXSample/ViewModel/MainViewModel.cs
using DevExpress.Mvvm;
using DevExpress.Mvvm.POCO;

namespace DXSample.ViewModel {
    public class MainViewModel {
        private INavigationService NavigationService { get { return this.GetService<INavigationService>(); } }

        public MainViewModel() {  }

        public void OnViewLoaded() {
            NavigationService.Navigate("HomeView", null, this);
        }
    }
}

File: VB/DXSample/ViewModel/MainViewModel.vb
Imports Microsoft.VisualBasic
Imports System
Imports DevExpress.Mvvm
Imports DevExpress.Mvvm.POCO

Namespace DXSample.ViewModel
	Public Class MainViewModel
		Private ReadOnly Property NavigationService() As INavigationService
			Get
				Return Me.GetService(Of INavigationService)()
			End Get
		End Property

		Public Sub New()
		End Sub

		Public Sub OnViewLoaded()
			NavigationService.Navigate("HomeView", Nothing, Me)
		End Sub
	End Class
End Namespace
